kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -671,7 +674,6 @@ window.TopicContainerOuterView = BaseView.extend({
     },
 
     back_to_parent: function() {
-        this.remove_topic_views(1);
```

Comment:
This is the important change here. The rest is just cleanup of unused back button code.